roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -88,14 +88,90 @@
     name: 'install_yamls_makes'
     tasks_from: 'make_edpm_baremetal_compute'
 
-- name: Deploy EDPM with baremetal
+- name: Prepare OpenStack Dataplane NodeSet CR
   vars:
-    make_edpm_deploy_baremetal_env: "{{ cifmw_edpm_deploy_baremetal_common_env |
+    make_edpm_deploy_baremetal_prep_env: "{{ cifmw_edpm_deploy_baremetal_common_env |
       combine(cifmw_edpm_deploy_baremetal_make_openstack_env)}}"
-    make_edpm_deploy_baremetal_dryrun: "{{ cifmw_edpm_deploy_baremetal_dry_run }}"
+    make_edpm_deploy_baremetal_prep_dryrun: "{{ cifmw_edpm_deploy_baremetal_dry_run }}"
   ansible.builtin.include_role:
     name: 'install_yamls_makes'
-    tasks_from: 'make_edpm_deploy_baremetal'
+    tasks_from: 'make_edpm_deploy_baremetal_prep'
```

Comment:
This make target does not add the repo-setup service in install_yamls https://github.com/openstack-k8s-operators/install_yamls/pull/809